app/controllers/admins/company_values_controller.rb

```diff
@@ -22,23 +22,23 @@ def create
       new_company_value = CompanyValue.new(company_value_params)
 
       if new_company_value.save
-        redirect_to admins_company_values_path, notice: 'New Company Value was successfully created.'
+        redirect_to admins_company_values_path, info: 'New Company Value was successfully created.'
       else
         render :new, locals: { company_value: new_company_value }
       end
     end
 
     def update
       if company_value.update(company_value_params)
-        redirect_to admins_company_value_path(company_value), notice: 'Company Value was successfully updated.'
+        redirect_to admins_company_value_path(company_value), info: 'Company Value was successfully updated.'
```

Comment:
I'd keep notice and alert as they come as default. Given you don't work on the project alone - how other developers could know that you introduced a new flash message type? How about other controllers?